db/seeds.rb

```diff
@@ -12,39 +12,65 @@ def time_floor(time, seconds = 15.minutes)
   Time.at((time.to_f / seconds).floor * seconds).utc
 end
 
-admin = User.new(admin: true, username: "admin", name: Faker::Name.name, email:"admin@spacebook.com", phone_num: Faker::PhoneNumber.phone_number, profile_pic: Faker::Avatar.image, password:"password")
+google = Company.new(name: "Google")
+google.save!
+
+valve = Company.new(name: "Valve")
+valve.save!
+
+admin = User.new(admin: true, username: "admin", name: Faker::Name.name, company_id: google.id, email:"admin@spacebook.com", phone_num: Faker::PhoneNumber.phone_number, profile_pic: Faker::Avatar.image, password:"password")
 admin.save!
 
 
-not_admin = User.new(admin: false, username: "not_admin", name: Faker::Name.name, email:"not_admin@spacebook.com", phone_num: Faker::PhoneNumber.phone_number, password:"password")
+not_admin = User.new(admin: false, username: "not_admin", name: Faker::Name.name, company_id: google.id, email:"not_admin@spacebook.com", phone_num: Faker::PhoneNumber.phone_number, password:"password")
+not_admin.save!
+
+
+4.times do
+  campus = Campus.new(company_id: google.id, city:Faker::Address.city, state:Faker::Address.state, zip:Faker::Address.zip, name:Faker::Name.name, picture_url: Faker::Avatar.image)
+  campus.save!
+end
+
+campus_user = CampusUser.new(user_id: admin.id, campus_id: google.campuses.sample.id)
+campus_user.save!
+
+campus_user = CampusUser.new(user_id: not_admin.id, campus_id: 1)
+campus_user.save!
+
+
+admin = User.new(admin: true, username: "admin", name: Faker::Name.name, company_id: valve.id, email:"vadmin@spacebook.com", phone_num: Faker::PhoneNumber.phone_number, profile_pic: Faker::Avatar.image, password:"password")
+admin.save!
+
+not_admin = User.new(admin: false, username: "not_admin", name: Faker::Name.name, company_id: valve.id, email:"vnot_admin@spacebook.com", phone_num: Faker::PhoneNumber.phone_number, password:"password")
 not_admin.save!
 
-10.times do
-  campus = Campus.new(city:Faker::Address.city, state:Faker::Address.state, zip:Faker::Address.zip, name:Faker::Name.name, picture_url: Faker::Avatar.image)
+4.times do
+  campus = Campus.new(company_id: valve.id, city:Faker::Address.city, state:Faker::Address.state, zip:Faker::Address.zip, name:Faker::Name.name, picture_url: Faker::Avatar.image)
   campus.save!
 end
 
-campus_user = CampusUser.new(user_id: admin.id, campus_id: rand(1..Campus.count))
+campus_user = CampusUser.new(user_id: admin.id, campus_id: valve.campuses.sample.id)
 campus_user.save!
 
 campus_user = CampusUser.new(user_id: not_admin.id, campus_id: 1)
 campus_user.save!
 
 
-30.times do
-  user = User.new(admin: false, username: Faker::Internet.user_name, name: Faker::Name.name, email:Faker::Internet.email, phone_num: Faker::PhoneNumber.phone_number, profile_pic: Faker::Avatar.image, password:"password")
+200.times do
```

Comment:
with 30, some campuses only had 0-2 users, so I scaled it up - I can tone it down to 100 though